src/app/features/collectibles/components/collectible-hover.tsx

```diff
@@ -20,11 +25,30 @@ export function CollectibleHover(props: { hoverText?: string }) {
       width="100%"
       zIndex={999}
     >
-      {hoverText ? (
-        <Text bottom="0px" color="white" left="0px" lineHeight="1.5" p="base" position="absolute">
-          {hoverText}
-        </Text>
-      ) : null}
+      <Box position="absolute" left="12px" bottom="12px">
+        {collectibleTypeIcon}
+      </Box>
+      {onClickCallToAction && (
+        <Flex
+          onClick={e => {
+            e.stopPropagation();
+            onClickCallToAction();
+          }}
+          tabIndex={-1}
```

Comment:
I'll ask for more design clarity around this, because I was trying to prevent having 2 focusable elements with the same action: clicking on the card itself or this button do the same thing

Reply:
Clarification from design: we'll use the CTA button & the send button as only two buttons on the card for now, so will set this to `as="button"`.